packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/KeyValuesTable.css

```diff
@@ -40,6 +40,18 @@ limitations under the License.
   width: 125px;
 }
 
+.KeyValueTable--copyColumn {
+  text-align: right;
+}
+
+.KeyValueTable--copyIcon {
+  visibility: hidden;
+}
+
+tr:hover > .KeyValueTable--copyColumn > .KeyValueTable--copyIcon {
```

Comment:
Maybe a class should be added for the row? That will shorten the other selectors and avoid use of the element selector. Maybe the `KeyValueTable--body` class can be removed.

Reply:
added `.KeyValueTable--row` and remove references to `tr` from .css file. still need `KeyValueTable--body` for vertical alignment of table itself.